build.zig

```diff
@@ -154,16 +147,9 @@ fn buildWhisper(b: *std.Build, args: struct {
             "-DGGML_METAL_EMBED_LIBRARY=OFF",
             "-DGGML_METAL=OFF",
         });
-
-    if (args.target.result.isMinGW()) {
-        whisper_configure.addArgs(&.{
-            "-G",
-            "MinGW Makefiles",
-        });
-    }
     if (args.target.result.os.tag == .linux or args.target.result.os.tag == .windows)
         whisper_configure.addArgs(&.{
-            "-DBUILD_SHARED_LIBS=ON",
```

Comment:
undo and remove `or args.target.result.os.tag == .windows` on if. linux shared-lib keep.

Reply:
changed